OpenRA.Game/Widgets/WidgetUtils.cs

```diff
@@ -158,14 +158,14 @@ public static void DrawPanelPartial(string collection, Rectangle bounds, PanelSi
 				DrawRGBA(cornerBottomRight, new float2(bounds.Right - cornerBottomRight.Size.X, bounds.Bottom - cornerBottomRight.Size.Y));
 		}
 
-		public static string FormatTime(int ticks)
+		public static string FormatTime(int ticks, int timestep = 40)
```

Comment:
Better not allow those overloads with magic values and instead fix all occurrences.